### PR TITLE
add opcode methods to make it easier to construct instruction scripts.

### PR DESCRIPTION
**Pull request by Jim8y**

### CODE REVIEW

File: src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+    private Instruction ReverseN(int? count)
+    {
+        if (count.HasValue)
+            Push(count.Value);
+        return AddInstruction(OpCode.REVERSEN);
+    }
+
+    #endregion
+
+
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs:128]:
```suggestion
```

File: src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+    }
+
+    private Instruction PackStruct()
+    {
+        return AddInstruction(OpCode.PACKSTRUCT);
+    }
+
+    private Instruction Pack(int? size)
+    {
+
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs:377]:
```suggestion
```

File: src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+        if (size.HasValue)
+            Push(size.Value);
+        return AddInstruction(OpCode.PACK);
+
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert/StackHelpers.OpCodes.cs:381]:
```suggestion
```

**Comment by shargon** [comment]
Change the previous calls?

**Comment by Jim8y** [comment]
> Change the previous calls?

Will work on new syscalls and see the experience,,,,,,

**Comment by shargon** [comment]
> > Change the previous calls?
> 
> Will work on new syscalls and see the experience,,,,,,

I don't understand the addiction of new methods if they are not used

**Comment by Jim8y** [comment]
> > > Change the previous calls?
> > 
> > Will work on new syscalls and see the experience,,,,,,
> 
> I don't understand the addiction of new methods if they are not used

its ok if you dont like it, not a necessary pr, it just same my effort to add new syacalls. no need to write addinatruction, but directly use the corresponding opcode methods.

**Review by shargon — APPROVED** [review]

**Comment by Jim8y** [comment]
fix conflict

**Comment by Jim8y** [comment]
please review again, there was a conflic and i have to update it. @shargon

**Review by shargon — DISMISSED** [review]

**Comment by Jim8y** [comment]
@shargon I am crying on it,,,, conflict again,,,, i will break prs into even smaller if you dont mind,,,,, too many conflicts now.

**Review by shargon — DISMISSED** [review]

**Comment by shargon** [comment]
It doesn't build..

**Review by shargon — APPROVED** [review]

**Pull request merged**